PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -631,14 +631,14 @@ def getCorrectedMET(self, process, metType, correctionLevel,produceIntermediateC
         #Txy parameter tuning
         if "Txy" in correctionLevel:
             self.tuneTxyParameters(process, corScheme, postfix)
-            getattr(process, "patPFMetTxyCorr"+postfix).srcPFlow = self._parameters["pfCandCollection"].value
+            getattr(process, "patPFMetTxyCorr"+postfix).srcPFlow = "puppiForMET" if self.getvalue("Puppi") else self._parameters["pfCandCollection"].value
```

Comment:
use of a syntax
self._parameters["pfCandCollection"].value

Reply:
this is the same as `self.getvalue("pfCandCollection")`